docs/resources/ADR-51/size.md

```diff
@@ -4,7 +4,11 @@ Validates that the full request size is within limits
 
 ```
 parameter ADR_45_TIMESTAMP = 1648771200
-
+parameter MAX_SIZES_BY_ENTITY_TYPE:
+  - SCENE: 15MB
```

Comment:
Do scenes have variable limits on the size? The main plaza weights around 200MB. It also has a hundred parcels. Does the validation takes that into account?

Reply:
Added clarification that the size is per pointer (parcel in scenes).